app/models/job.ts

```diff
@@ -439,12 +441,13 @@ export class Job extends Record {
   }
 
   /**
-   * Returns only the links with a rel that matches the passed in value
+   * Returns only the links with a rel that matches the passed in value.
    *
    * @param rel - the relation to return links for
    * @returns the job output links with the given rel
    */
   getRelatedLinks(rel: string): JobLink[] {
-    return this.links.filter((link) => link.rel === rel);
+    const links = this.links.filter((link) => link.rel === rel);
```

Comment:
I can't recall quite how / where this is used, but I'm expecting it to need to be changed to do an actual query once pagination comes into play and we can't just store all of the links in an array.

Reply:
Oh... I think this is mostly used in STAC catalogs... how do they handle pagination?

Reply:
They don't yet. Mostly just an observation that we can't really rely on storing all links in memory or we lose out on the benefits of pagination.  This can wait for those tickets but needs to be addressed there.